crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -629,6 +635,8 @@ enum ValidatorUpdateErrorKind {
     Secp256k1NotSupported,
     #[error("bytes stored in the .pub_key field could not be read as an ed25519 verification key")]
     VerificationKey { source: crate::crypto::Error },
+    #[error("validator name was {length} characters long, but must be 32 at most")]
```

Comment:
Would be good to define the value as a `const` so we can use it in this error string and in the actual check of `name.len()` so they don't fall out of sync in the uture.

Reply:
Made a constant for use in both the error message and for the check itself in 0939a6e.